lib/index.js

```diff
@@ -9,6 +9,7 @@ const PLUGIN_NAME = 'gulp-rollup-each'
 const cache = {} // cache - (outside of export)
 
 module.exports = function (arg1, arg2, injectedRollup) {
+
```

Comment:
Is it your style to put blank lines on the first line?
I would like to leave it as it is without putting empty lines if possible.
I often use JavaScript Standard Style.

Reply:
Hey @ko-yelie –  I generally have ESLint and prettier handling formatting. Maybe worth adding in your style preferences with a `.rc` file.

Reply:
Great! Always enjoy working on this poetic little plugin. Happy New Year friend.